### DISCUSSION TITLE
Where can I find a description for the differences in the settings

### DISCUSSION THREAD

**Discussion by bignarc** [Cookie-AutoDelete/Cookie-AutoDelete#1218] [Q&A / Support] [answered]
Hi, I apologize in advance because I'm sure this information is out there somewhere I just can't find it.

I have a very basic understanding of cookies.

![image](https://user-images.githubusercontent.com/89874357/131571611-2824d960-5670-4a24-a40d-738e5dc043bc.png)

Can I get any sort of summarized explanation of what these options are?

**Comment by ghost** [comment, marked as answer]
https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/Documentation#other-browsing-data-cleanup-options

**Reply by kennethtran93** [reply]
Thanks @gwarser 

To add some additional info here, the link above will contain all those options / site data, with additional links to those specific site data types.
